Resources/Prototypes/Entities/Structures/object_sensor.yml

```diff
@@ -0,0 +1,41 @@
+- type: entity
+  id: ObjectSensor
```

Comment:
inherit the base logic gate to cut down on dupe components

Reply:
BaseLogicItem inherit from BaseItem, which i do not want

Reply:
would be inconsistent with every other logic thing
if it doesnt work when unanchored then being an item has no downside

Reply:
> would be inconsistent with every other logic thing if it doesnt work when unanchored then being an item has no downside

it's not supposed to be a logic thing